turbo.json

```diff
@@ -3,7 +3,9 @@
   "globalEnv": [
     "GAID",
     "NEXT_PUBLIC_APTOS_API_KEY_TESNET",
-    "NEXT_PUBLIC_APTOS_API_KEY_DEVNET"
+    "NEXT_PUBLIC_APTOS_API_KEY_DEVNET",
+    "NEXT_PUBLIC_SWAP_CCTP_MAIN_SIGNER_PRIVATE_KEY",
+    "NEXT_PUBLIC_SWAP_CCTP_SPONSOR_ACCOUNT_PRIVATE_KEY"
```

Comment:
are these needed in the global turbo.json? is there any way to move them under the specific app they're used in?

Reply:
turbo needs to be aware of it when the dapp is deployed to a live server

Reply:
Yeah I understand, but if you defined them under the example app's `turbo.json` it would also work right?
So it's a little more clearer that they're only related to that specific application.

Reply:
don't worry about it, we can figure it out later whether that's possible

Reply:
Just for reference:
https://turbo.build/repo/docs/reference/package-configurations